schema/field/numeric.go

```diff
@@ -1474,6 +1537,13 @@ func (b *uint64Builder) Default(i uint64) *uint64Builder {
 	return b
 }
 
+// DefaultFunc sets the function that is applied to set the default value
+// of the field on creation.
+func (b *uint64Builder) DefaultFunc(fn interface{}) *uint64Builder {
+	b.desc.Default = fn
+	return b
+}
+
```

Comment:
Please add a check for the function signature in the `Descriptor` method as exists in other builders (e.g. see [`stringBuilder`](https://github.com/facebook/ent/blob/master/schema/field/field.go#L297-L302)).